factory/factory.Dockerfile

```diff
@@ -29,7 +29,7 @@ ENV DBUS_SESSION_BUS_ADDRESS=/dev/null \
 # we really only need to worry about the top folder, fortunately
 # TODO: there are other folders that need permissions but I don't know what they are yet, See: https://github.com/cypress-io/cypress/issues/23962
 RUN ls -la /root \
-  && chmod 755 /root \
+  && chmod 777 /root \
```

Comment:
Does this address https://github.com/cypress-io/cypress/issues/23962, linked in the comment above?

Reply:
No, this doesn't address that issue. I had hoped it would but something else needs to change to fix that issue. This, however is a solid step forward and at least gets us back to supporting the node user.

I'll have to dig further to solve the linked issue. (Thanks for bringing this up, i couldn't remember where that issue was. I should have read the comment!)